cleverhans/experimental/certification/dual_formulation.py

```diff
@@ -303,31 +357,88 @@ def get_full_psd_matrix(self):
         axis=0)
     return self.matrix_h, self.matrix_m
 
-  def compute_certificate(self):
+  def make_M_psd(self):
```

Comment:
it's better to call this function `make_m_psd` or `make_m_matrix_psd`
Usually functions in Python are names using `all_lower_case_with_underscore' or 'CamelCase' naming convention. and naming convention should be consistent across the codebase.

Take a look at Python style naming convention as well: https://www.python.org/dev/peps/pep-0008/#naming-conventions

It looks like Cleverhans linter does not enforce naming convention very well